homeassistant/components/climate/mysensors.py

```diff
@@ -71,15 +71,21 @@ def target_temperature(self):
     def target_temperature_high(self):
         """Return the highbound target temperature we try to reach."""
         set_req = self.gateway.const.SetReq
-        if set_req.V_HVAC_SETPOINT_COOL in self._values:
+        if set_req.V_HVAC_SETPOINT_COOL in self._values and \
+                set_req.V_HVAC_SETPOINT_HEAT in self._values:
             return self._values.get(set_req.V_HVAC_SETPOINT_COOL)
+        else:
+            return None
```

Comment:
You don't need to both check that the key is in the dict and use the dict.get method. Also dict.get will return None by default if it doesn't find the key that you pass as first argument. This is enough:
```py
def target_temperature_high(self):
    set_req = self.gateway.const.SetReq
    if set_req.V_HVAC_SETPOINT_HEAT in self._values:
        return self._values.get(set_req.V_HVAC_SETPOINT_COOL)
```